packages/react/src/box/styles.ts

```diff
@@ -269,6 +269,12 @@ type LayoutProps = Partial<{
 	alignItems: ResponsiveProp<
 		'stretch' | 'flex-start' | 'flex-end' | 'center' | 'baseline'
 	>;
+	/** Sets the CSS align-self property.
+	 * @see https://developer.mozilla.org/en-US/docs/Web/CSS/align-self
+	 */
+	alignSelf: ResponsiveProp<
```

Comment:
This needs to be added to a changeset

Reply:
- [ ] Add `alignSelf` to a changeset

Reply:
There are many other uses of `css={{ alignSelf: 'flex-start' }}`. Perhaps we should change them?